pcsx2-qt/Settings/SettingsDialog.h

```diff
@@ -51,7 +51,7 @@ class SettingsDialog final : public QDialog
 
 public:
 	explicit SettingsDialog(QWidget* parent);
-	SettingsDialog(QWidget* parent, std::unique_ptr<SettingsInterface> sif, const GameList::Entry* game, std::string serial, u32 disc_crc);
+	SettingsDialog(QWidget* parent, std::unique_ptr<SettingsInterface> sif, const GameList::Entry* game, std::string serial, u32 disc_crc, QString filename = QString());
```

Comment:
`const QString&`

Reply:
I'm `std::move`-ing it in so it's intended, same as `std::string serial`.